patches/electron-context-menu+4.0.4.patch

```diff
@@ -103,3 +103,12 @@ index b10daea..ea2f891 100644
  				},
  			}),
  			copyImage: decorateMenuItem({
+@@ -350,7 +349,7 @@ const create = (win, options) => {
```

Comment:
@devinbinnie can you give a bit on context on what is this file we are changing?

Reply:
It's the package `electron-context-menu`, it provides the right-click menu used in some places.

Reply:
No this is my patch I made to fix the issue.

Reply:
Is a divergence to the upstream module? I'm just trying to understand the process

Reply:
Yes. They have a bug where they assume that the object passed in has an `isDestroyed` method. By consequence, the ones they've defined do, but the `WebContentsView` that we're using does not, it's supposed to be on the `webContents` object. They're misusing the schema of where the function is supposed to be, so I'm correcting it with this patch.